extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.instant.js

```diff
@@ -41,7 +39,8 @@ require([
 			plist: recommendedPlaylist,
 			vtags: videoTags
 		},
-		responseTimeout = 2000,
+		responseTimeout = 3000,
```

Comment:
Why 3000?

Reply:
Cause it's more than 2000 :D This scenario is probably only for devbox environments, because it's not likely to fully disable Prebid on production, so I left it as a hack. But after `waitForResponse` refactor it won't be needed.